tools/topology/sof-byt-codec.m4

```diff
@@ -98,5 +98,9 @@ VIRTUAL_WIDGET(ssp2 Rx, out_drv, 1)
 VIRTUAL_WIDGET(ssp2 Tx, out_drv, 2)
 VIRTUAL_WIDGET(ssp0 Tx, out_drv, 3)
 VIRTUAL_WIDGET(ssp0 Rx, out_drv, 4)
-VIRTUAL_DAPM_ROUTE_IN(modem_in, SSP, 0, IN, 0)
-VIRTUAL_DAPM_ROUTE_OUT(modem_out, SSP, 0, OUT, 1)
+VIRTUAL_WIDGET(modem_out, out_drv, 5)
+VIRTUAL_WIDGET(modem_in, input, 6)
+VIRTUAL_WIDGET(AIF2 Playback, out_drv, 7)
+VIRTUAL_WIDGET(AIF2 Capture, input, 8)
+VIRTUAL_WIDGET(AIF1 Playback, out_drv, 9)
+VIRTUAL_WIDGET(AIF1 Capture, input, 10)
```

Comment:
no, those AIF thingies are RT5640 widgets, we need to leave them also.

Also why is all this out_drv? isn't there an in_drv?

and last should we move all this to the byt/cht.m4 files?